build/wix/Localization/README-Translations.md

```diff
@@ -1,66 +0,0 @@
-
-Transifex doesn't handle wxl files.
```

Comment:
I think we still need these files, right?

Reply:
No, localized WIX installers are a total hack and I wasted way too much time to make them work with CPack. If nobody volunteers to implement it, let's remove them. We could just pull them from transifex if somebody wants to add them, right?

Reply:
What is the current situation?
Scons = localized wix 
Cmake = english only?